docs/release-notes/1.5.0-preview2/release-1.5.0-preview2.md

```diff
@@ -0,0 +1,62 @@
+# [ML.NET](http://dot.net/ml) 1.5.0-preview2
+
+## **New Features (IN-PREVIEW, please provide feedback)**
+- **TimeSeriesImputer** ([#4623](https://github.com/dotnet/machinelearning/pull/4623))
+- **LDSVM Trainer** ([#4060](https://github.com/dotnet/machinelearning/pull/4060))
```

Comment:
>LDSVM Traine [](start = 4, length = 12)

Describe what this trainer is and where it is used and then provide links to samples.

Reply:
Description:
LDSVM uses trees as its SVM kernel to create a non-linear binary learner.

Sample:
https://github.com/dotnet/machinelearning/blob/c819d77e9250c68883713d5f1cd79b8971a11faf/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/LdSvm.cs

Reference links:
http://manikvarma.org/pubs/jose13.pdf
http://manikvarma.org/talks/ICML13.pdf

Reply:
thanks and in addition I would also
expand LDSVM acronym.

Reply:
"Local Deep SVM"